validator/src/main/resources/expected-data-template/metric_limiter/metric-limiter-metric.mustache

```diff
@@ -0,0 +1,92 @@
+-
+  metricName: Latency
+  namespace: {{metricNamespace}}
+  dimensions:
+    -
+      name: Operation
+      value: AllOtherOperations
```

Comment:
Do we still have logics to verify any operation escaped from metrics limiting? The threshold we set is still 2, right?

Reply:
The threshold is still 2. We still send two irrelevant API calls to the sample app initially to make it so that we exceed the threshold.